### PR TITLE
[T] Update Illustrations (big icons)

### PR DESCRIPTION
**Pull request by viniciuskneves**
Previous PR: https://github.com/homeday-de/homeday-blocks/pull/473

---

- Figma file updated (Using the new one from the design team)
- Name regex updated
  - Before we had the following situation: `ic_icon-name_something` would become `iconName` and `ic_icon-name` would become `null`
  - Now the `ic_icon-name` becomes `iconName` as expected (once the PR is approved and merged I will revisit the [docs](https://homeday.atlassian.net/wiki/spaces/FRO/pages/691208197/Assets+Icons) and update it as the case `ic_icon-name` is not clear that it wouldn't work)
  - I dropped the lookahead in the regex and opted for a "simpler" version of it, please, let me know if I missed something
- Removed two illustrations that were removed (please, double check if your project is using it but I can imagine it isn't as the illustrations were revisited by the designers)

---

*If we agree on removing the illustrations above, I will update the importer script to do it automatically. Currently it doesn't deleted the old illustrations.*

### CODE REVIEW

File: src/assets/big-icons/ic_bell-reminder.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...25.955 66.0999 121.68 68.6263 118.934C68.5551 118.708 68.379 118.324 67.9987 117.752C67.0823 116.375 65.4399 114.571 63.231 112.312C62.3253 111.386 61.3366 110.395 60.2901 109.347C56.3454 105.396 51.5795 100.623 47.342 95.3571C41.9846 88.6996 37.2459 80.9939 36.0011 72.7523ZM72.5902 118.32C72.7938 119.4 72.6036 120.604 71.6602 121.548C70.4886 122.719 70.4886 124.619 71.6602 125.79L76.6099 130.74C77.391 131.521 78.6573 131.521 79.4384 130.74L128.936 81.2427C129.717 80.4617 129.717 79.1953 128.936 78.4143L123.986 73.4645C122.815 72.293 120.915 72.293 119.743 73.4645C118.843 74.365 117.713 74.7653 116.517 74.719C115.413 74.6762 114.343 74.2594 113.375 73.7316C111.448 72.6809 109.277 70.8195 106.992 68.6668C105.443 67.2074 103.809 65.5848 102.086 63.8741C98.5574 60.3709 94.6572 56.4984 90.3535 52.9022C83.9809 47.5771 77.111 43.2153 70.0616 42.0027C63.1832 40.8195 55.8667 42.589 48.3257 50.13C40.7825 57.6732 38.8931 65.116 39.9563 72.1549C41.0427 79.3481 45.2533 86.3812 50.4583 92.8494C54.5465 97.9296 59.1086 102.5 63.0415 106.44C64.112 107.513 65.1359 108.539 66.0909 109.515C68.2737 111.748 70.1751 113.802 71.329 115.536C71.8992 116.393 72.4062 117.344 72.5902 118.32Z" fill="#1C3553"/>
```
Comment (LarsFe) [inline src/assets/big-icons/ic_bell-reminder.svg:5]: Oh ohhh. There is an error inside. It should be:

```
<path fill-rule="evenodd" clip-rule="evenodd" d="M36.0011 72.7523C34.733 64.3564 37.1305 55.6683 45.4972 47.3016C53.8661 38.9327 62.4595 36.6363 70.7397 38.0606C78.8489 39.4555 86.3863 44.3744 92.9184 49.8327C97.3473 53.5336 101.493 57.6479 105.086 61.2131C106.766 62.88 108.325 64.4269 109.735 65.7553C112.024 67.9115 113.864 69.4421 115.29 70.2199C115.999 70.6064 116.441 70.713 116.672 70.722C116.813 70.7274 116.843 70.7079 116.915 70.6361C119.649 67.9024 124.081 67.9024 126.815 70.6361L131.764 75.5859C134.107 77.929 134.107 81.728 131.764 84.0711L82.2668 133.569C79.9236 135.912 76.1247 135.912 73.7815 133.569L68.8318 128.619C66.1684 125.955 66.0999 121.68 68.6263 118.934C68.5551 118.708 68.379 118.324 67.9987 117.752C67.0823 116.375 65.4399 114.571 63.231 112.312C62.3253 111.386 61.3366 110.395 60.2901 109.347C56.3454 105.396 51.5795 100.623 47.342 95.3571C41.9846 88.6996 37.2459 80.9939 36.0011 72.7523ZM72.5902 118.32C72.7938 119.4 72.6036 120.604 71.6602 121.548C70.4886 122.719 70.4886 124.619 71.6602 125.79L76.6099 130.74C77.391 131.521 78.6573 131.521 79.4384 130.74L128.936 81.2427C129.717 80.4617 129.717 79.1953 128.936 78.4143L123.986 73.4645C122.815 72.293 120.915 72.293 119.743 73.4645C118.843 74.365 117.713 74.7653 116.517 74.719C115.413 74.6762 114.343 74.2594 113.375 73.7316C111.448 72.6809 109.277 70.8195 106.992 68.6668C105.443 67.2074 103.809 65.5848 102.086 63.8741C98.5574 60.3709 94.6572 56.4984 90.3535 52.9022C83.9809 47.5771 77.111 43.2153 70.0616 42.0027C63.1832 40.8195 55.8667 42.589 48.3257 50.13C40.7825 57.6732 38.8931 65.116 39.9563 72.1549C41.0427 79.3481 45.2533 86.3811 50.4583 92.8494C54.5465 97.9296 59.1086 102.5 63.0415 106.44C64.112 107.513 65.1359 108.539 66.0909 109.515C68.2737 111.748 70.1751 113.802 71.329 115.536C71.8992 116.393 72.4062 117.344 72.5902 118.32Z" fill="#1C3553"/>
```

Comment (LarsFe) [reply src/assets/big-icons/ic_bell-reminder.svg:5]: 🙈 All good.

**Comment by LarsFe** [comment]
The two SVGs that you mentioned, which are removed aren't used in customer-app. The PR looks fine for me so far. 

And woooow, these are lots of changes. Are they going to be QAed? (Actually this would need to be done.) Maybe in the future it's better to separate such big PRs into multiple, that it can be checked easier – step by step? 76 changes is quite a lot, don't want to talk about the linked PR in the description. 🙈
Man I wish I would have had home office at Lesara. Creating such PRs was dangerous in previous times. Our QA lady set across from me. Stopping her from jumping over the table was only possible by giving her sweets. 😀

**Comment by ilyasmez** [comment]
> The two SVGs that you mentioned, which are removed aren't used in customer-app. The PR looks fine for me so far.

I've checked customer-aap and realtor-app, they are not used there, but nonetheless, we should bump a major for this release.

> And woooow, these are lots of changes. Are they going to be QAed? (Actually this would need to be done.) Maybe in the future it's better to separate such big PRs into multiple, that it can be checked easier – step by step? 76 changes is quite a lot, don't want to talk about the linked PR in the description. 🙈

I've not sure if somebody already introduced you to the assets importer we are using 🤔 
We usually just run the command and create a PR with the changes, because it just syncs Blocks with the Figma file that designers are maintaining (so it's pre-QAed :D as I believe designers are the eligible ones to QA these kind of UI changes)

**Pull request merged**